src/components/art/ArtList.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ArtItem from './ArtItem';
+
+export default function ArtList({ drawings }) {
+    return (
+        <section>
+            {drawings.map((art) => {
+                return (
+                    <label key={art.id}>
```

Comment:
Do we want to use UL or MUI List components here? Definitely, something we can refactor later instead so we can just get a working prototype up.